backend/src/test/java/com/google/sps/EmailNotifierTest.java

```diff
@@ -68,62 +67,69 @@ public MimeMessage convertToMimeMessage(Message msg) throws MessagingException {
   }
 
   @Test
-  public void testMessageHasCorrectApplicationName()
-      throws MessagingException, IOException, GeneralSecurityException {
+  public void testMessageHasCorrectApplicationName() throws MessagingException, IOException {
```

Comment:
here too

Reply:
What does here too refer to exactly ?

Reply:
My bad, I think I misread this when I left the comment.